eslint.config.js

```diff
@@ -28,6 +28,13 @@ export default ts.config(
         },
     },
     {
-        ignores: ["build/", ".svelte-kit/", "dist/", "coverage/", "node_modules/"],
+        ignores: [
+            "build/",
+            ".svelte-kit/",
+            "dist/",
+            "coverage/",
+            "node_modules/",
+            "src/lib/model/api/",
```

Comment:
While we currently don't have any files in `api/` to lint, we should add it for the future.
```suggestion
            "src/lib/model/api/",
            "api/",
```

Reply:
I agree.